common/test/utils/works.test.js

```diff
@@ -5,8 +5,9 @@ import {
   getItemsWith,
   getItemIdentifiersWith,
   getWorkIdentifiersWith,
+  getArchiveAncestorArray,
 } from '../../utils/works';
-import { WorkFixture } from '../fixtures/catalogueApi/work';
+import { WorkFixture, workWithPartOf } from '../fixtures/catalogueApi/work';
```

Comment:
We're using different capitilisations here, is there a reason?

Reply:
no reason, I'll make it consistent